metricbeat/module/elasticsearch/index/data_xpack.go

```diff
@@ -120,6 +121,14 @@ type shardStats struct {
 	Relocating   int `json:"relocating"`
 }
 
+type bulkStats struct {
+	TotalOperations   int `json:"total_operations"`
+	TotalTimeInMillis int `json:"total_time_in_millis"`
+	TotalSizeInBytes  int `json:"total_size_in_bytes"`
+	AvgTimeInMillis   int `json:"throttle_time_in_millis"`
```

Comment:
@ycombinator Is this a typo? Should it be `json:"avg_time_in_millis"`?

Reply:
Yes, it should 😞. Good eye, nice catch! I assume parity tests failed? I'll put up a PR to fix it. Thanks!

Reply:
PR is up: https://github.com/elastic/beats/pull/18420